mautic/mautic
# Format lifecycle charts

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | Y 
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | na
| BC breaks? | na
| Deprecations? | n

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:
This PR adds a new email widget for devices used to read emails, and fixes the hover on devices for the lifecycle chart

#### Steps to test this PR:
1. Apply PR
2. look at the lifecycle chart and devices should have a hover that can be read
3. Add new devices for emails read widget and it should give you a pie chart of devices used to read emails

### As applicable
#### Steps to reproduce the bug:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
